recipes/plantcv/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+version=$($PYTHON setup.py version | grep Version | sed -e 's/Version: //')
+sed -i'' -e "s/versioneer.get_version()/'$version'/g" setup.py
+sed -i'' -e 's/cmdclass=versioneer.get_cmdclass(),//g' setup.py
+sed -i'' -e 's/setup_requires=\["pytest-runner"\],//g' setup.py
+sed -i'' -e "s/tests_require=\['pytest'\],//g" setup.py
```

Comment:
Maybe just write a patch file?

Reply:
Thanks @scopatz, the patch file is a lot simpler!